net-wireless/bluetuith/bluetuith-0.1.7.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+
+DESCRIPTION="A TUI bluetooth manager for Linux written in Go"
+HOMEPAGE="https://darkhz.github.io/bluetuith"
+
+GIT_DOCUMENTATION_COMMIT="9c215bb1a64bdbf0f88060db4de6701215799033"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/darkhz/bluetuith.git"
+else
+	SRC_URI="https://github.com/darkhz/bluetuith/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI+=" https://github.com/rahilarious/gentoo-distfiles/releases/download/${P}/deps.tar.xz -> ${P}-deps.tar.xz"
+	SRC_URI+=" https://github.com/darkhz/bluetuith/archive/${GIT_DOCUMENTATION_COMMIT}.tar.gz -> ${P}-docs.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="Apache-2.0 BSD-2 BSD MIT"
+SLOT="0"
+
+IUSE="doc"
+RESTRICT="test"
+RDEPEND="
+	net-wireless/bluez
+"
+
+if [[ ${PV} == *9999* ]]; then
+	src_unpack() {
+		git-r3_src_unpack
+		git clone --depth 1 -b documentation "${EGIT_REPO_URI}" "${PN}-${GIT_DOCUMENTATION_COMMIT}" || die
+		go-module_live_vendor
+	}
+fi
+
+src_configure() {
+	return
```

Comment:
Do you really need this, or is it fine if you don't specify it?

Reply:
I know but default src_configure runs configure if it exists. What if in future upstream adds that `configure` and it gets run without me knowing (never gonna happen) but still because this is live, playing it as safe as possible

Reply:
I feel it's the wrong kind of thinking. All sorts of stuff could go wrong then. What if they rename all their files? What if they migrate away from Go? What if they add a submodule we need to download...?

It just leads to noise in the ebuild and makes me think there's a configure script we're intentionally not running.

Reply:
Your logic checks out but I was just being cautious for security reasons (what if upstream gets hacked and sets malicious configure script 😂).

But Yeah will remove the noise.